pkg/cmd/admin/registry/registry.go

```diff
@@ -74,7 +75,14 @@ type RegistryConfig struct {
 
 var errExit = fmt.Errorf("exit")
 
-const defaultLabel = "docker-registry=default"
+const (
+	defaultLabel = "docker-registry=default"
+	defaultPort  = 5000
```

Comment:
There is an upstream type `IntOrStr` https://github.com/kubernetes/kubernetes/blob/master/pkg/util/intstr/intstr.go#L35-L39